srv/pom.xml

```diff
@@ -173,7 +173,7 @@
 							<goal>npm</goal>
 						</goals>
 						<configuration>
-							<arguments>install</arguments>
+							<arguments>install @sap/cds-dk@${cds.install-cdsdk.version} @sap/cds-mtxs@^1 --no-save</arguments>
```

Comment:
Is this just a workaround or should we improve the goal `install-cdsdk` in a way, that it also installs @sap/cds-mtxs in addition to the @sap/cds-dk? Maybe it can be derived from the project configuration if this dependency needs to be installed or not. Or with an additional switch.

Reply:
To me all of this should be solved by the @sap/cds-dk tooling itself. For me this is a workaround. Note that the normal @sap/cds-dk would also be fine if installed not from npmjs.org but from artifactory. Because then the node_modules layout is different and the tooling relies on the structure provided when installing from artifactory. We shouldn't built anything of this into the cds-maven-plugin until it is not 100% clear that this is the correct way forward.

Reply:
Mhh, really strange that it makes a difference if the cds-dk is installed from npmjs.org or from artifactory. I will never understand how npm works :-(.